src/plugin/memory-webview-main.ts

```diff
@@ -225,9 +225,10 @@ export class MemoryWebview implements vscode.CustomReadonlyEditorProvider {
         const visibleColumns = CONFIGURABLE_COLUMNS
             .filter(column => vscode.workspace.getConfiguration(manifest.PACKAGE_NAME).get<boolean>(column, false))
             .map(columnId => columnId.replace('columns.', ''));
+        const addressPadding = AddressPaddingOptions[memoryInspectorConfiguration.get(manifest.CONFIG_ADDRESS_PADDING, manifest.DEFAULT_ADDRESS_PADDING)];
```

Comment:
For consideration (but not necessarily for this PR): we could make this preference fetching / transmitting section less verbose if we used the same schema for the message as the preferences. The new could just do something like

```typescript
return vscode.workspace.getConfiguration().get(manifest.PACKAGE_NAME) as OurMessageInterface;
```

The downside would be coupling between the `package.json` declarations and our data formats. But there's already a lot of coupling, since we're just pulling out the fields declared in the `package.json` one by one anyway.

Reply:
Yes, that sounds reasonable. One consideration though is that the VS Code settings are very much user facing. For instance, the values of enums are directly showing up in the UI, it is not possible to assign user-facing labels for values. This is also why I added this conversion here to avoid showing the user just 'Min', 0, 32, 64.

We could of course pull this translation into the webview. I agree that the coupling probably doesn't get much worse by that. It is just a bit of separation of concern: the webview-main translates everything related to the VS Code extension into something that the rather VSCode-independent webview would understand. But this is very subtle.

I'd certainly be open to that. However, I feel it should go into a separate PR, if you agree. Should I open an issue on this matter?

Reply:
I've created an issue for (#90).